### PR TITLE
Changed any rgb definitions that used alphas to rgba.

### PR DESCRIPTION
**Pull request by pscadding**
We are getting styling warnings when using Workfiles in Photoshop or After Effects.

The warnings look like this:
`QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 0,0,0,0'`

I've updated all the places that define alpha values to use rgba, instead of just rgb, unless they were defining 255 in which case I just removed the alpha value.

### CODE REVIEW

File: python/tk_multi_workfiles/ui/open_options_form.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def setupUi(self, OpenOptionsForm):
 "border-style: none;\n"
 "border-width: 1px;\n"
 "border-color: rgb(0,0,0);\n"
-"background-color: rgb(255,255,255,48);\n"
+"background-color: rgba(255,255,255,48);\n"
```
[the remark this answers is not in the document]
Comment (pscadding) [reply python/tk_multi_workfiles/ui/open_options_form.py]: These are auto-generated files so I'm going to ignore them.

File: app.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def warning_color(self):
         :returns: An RGBA tuple of int (0-255).
         """
         color = sgtk.platform.qt.QtGui.QColor(self.style_constants["SG_ALERT_COLOR"])
-        return color.red(), color.green(), color.blue(), color.alpha()
```
Comment (pscadding) [inline app.py]: `SG_ALERT_COLOR` comes from tk-core and provides a hexadecimal color: `#FC6246` so no alpha.

File: python/tk_multi_workfiles/ui/open_options_form.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfboismenu) [reply python/tk_multi_workfiles/ui/open_options_form.py]: Hum... that's annoying another... it means our auto-generated files have the wrong data type for the color?

[the remark this answers is not in the document]
Comment (pscadding) [reply python/tk_multi_workfiles/ui/open_options_form.py]: Yeah I did edit the UI file and regenerated the build py file. Sorry should have called that out.

[the remark this answers is not in the document]
Comment (pscadding) [reply python/tk_multi_workfiles/ui/open_options_form.py]: The rgb, references were coming from style sheets that were defined in the ui file. I edited the file in PyCharm, but checked it was also changed and OK in designer.

**Pull request merged**